concourse/tasks/remove-healthcheck-db.yml

```diff
@@ -13,11 +13,17 @@ run:
     - |
       ./paas-cf/concourse/scripts/import_bosh_ca.sh
       . ./config/config.sh
-      if curl -I -f $API_ENDPOINT/info; then
-        echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
-        cf target -o testers -s healthcheck
-        cf delete healthcheck -f -r
-        if cf services | grep -q healthcheck-db; then
-          cf delete-service healthcheck-db -f
-        fi
+      if ! curl -I -f $API_ENDPOINT/info; then
+        echo "CF API unavailable. Skipping..."
+        exit 0
+      fi
+      echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
+      if ! cf org testers > /dev/null; then
```

Comment:
I'm unsure about this too..

It's seem that part of the problem is that we use this task in two different contexts:

- `autodelete-cloudfoundry` which is unattended and it's acceptable for the task to fail silently because the damage is limited
- `destroy-cloudfoundry` which is attended and it's not acceptable for this to fail silently because the instance will be orphaned - I think it's right that this pipeline expects a full environment to be created

Would this also be addressed by [#120765225: disable healthcheck DB in dev](https://www.pivotaltracker.com/story/show/120765225) because we'd no longer need to include this task in `autodelete-cloudfoundry`?